clients/src/test/java/org/apache/kafka/common/requests/RequestResponseTest.java

```diff
@@ -1098,7 +741,7 @@ public void testJoinGroupRequestVersion0RebalanceTimeout() {
 
     @Test
     public void testOffsetFetchRequestBuilderToStringV0ToV7() {
```

Comment:
We have `testJoinGroupRequestVersion0RebalanceTimeout` but `testOffsetFetchRequestBuilderToStringV0ToV7`, so rename `testJoinGroupRequestVersion0RebalanceTimeout` → `testJoinGroupRequestV0RebalanceTimeout` for consistency